testing/apache2-mod-security/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Ondrej Esler <ondrej.esler@intraworlds.com>
+# Maintainer: Ondrej Esler <ondrej.esler@intraworlds.com>
+pkgname=apache2-mod-security
+_pkgreal=modsecurity
+_realname=mod_security2
+pkgver=2.9.3
+pkgrel=0
+pkgdesc="Security Module for Apache2"
+url="https://www.modsecurity.org/"
+arch="all"
+license="Apache-2.0"
+depends="apache2 pcre libxml2"
+makedepends="autoconf automake libtool linux-headers apache2-dev pcre-dev libxml2-dev"
+subpackages="$pkgname-doc"
+source="https://github.com/SpiderLabs/ModSecurity/releases/download/v$pkgver/$_pkgreal-$pkgver.tar.gz
+	$pkgname.conf"
```

Comment:
Should the configuration file be named security.conf to follow the pattern in the main package?